docker/data/notebooks/Real_time_linear_regression.py

```diff
@@ -0,0 +1,123 @@
+#Connect to the client
```

Comment:
`docker/data` looks like it should not have been checked in.  It also looks like there should have been a `.gitignore` entry for it.

Reply:
Also, the `.whl` file should not have been committed.